src/physics/src/physics_factory_reacting_flows.C

```diff
@@ -202,7 +202,7 @@ namespace GRINS
       {
         {
           std::string msg = "WARNING: Specifying thermo_model = "+AntiochOptions::cea_nasa_model();
-          msg += "is DEPREACATED!\n";
+          msg += " is DEPREACATED!\n";
```

Comment:
Maybe fix the spelling while you're editing anyway?  :-D

Reply:
Wow, nice catch! It's fixed in the latest commit